src/plugins/intel_cpu/tests/functional/single_layer_tests/concat.cpp

```diff
@@ -118,23 +139,41 @@ const auto blocked16_5D_ref = CPUSpecificParams{{nCdhw16c}, {nCdhw16c}, {}, "ref
 const std::vector<ElementType> netPrecisions = {
         ElementType::i8,
         ElementType::i32,
+        ElementType::i64,
         ElementType::f32,
         ElementType::bf16
 };
 
+const ov::AnyMap emptyConfig = {};
```

Comment:
We have `CPUTestUtils::cpuEmptyPluginConfig`. Can we use it instead of creation empty config in each test file?